### ISSUE TITLE
Can't get user registration to work

### ISSUE THREAD

**Issue by SightSpirit** [moebooru/moebooru#70]
I'm setting up a local Moebooru server for personal enrichment, and after much struggle, I _finally_ got the application up and running. Unfortunately, after all that, I discovered that it's not processing my user registration. When I click the "Signup" button, it either cycles back to the signup page with no message, or it goes to /user/home with the message, "You are not logged in." Trying to log in also results in the latter.

I must have set something up wrong. What could be causing this issue?

**Comment by nanaya** [comment]
try setting `CONFIG["secure"]` to `false`

**Comment by SightSpirit** [comment]
That didn't work, unfortunately.

**Comment by andycontrib** [comment]
Same issue here. I made a [dockerized container of this app](https://github.com/andycontrib/moebooru-docker) so the issue can be completely recreate-able. If I followed your readme correctly, the correct setup order is to:
1. bundle install
1. init postgres
1. create moebooru user
1. bundle exec rake db:reset (using moebooru user)
1. bundle exec rake db:migrate (using moebooru user)
1. bundle exec unicorn
